pkg/share/manager/jsoncs3/jsoncs3_test.go

```diff
@@ -147,6 +150,58 @@ var _ = Describe("Jsoncs3", func() {
 		}
 	})
 
+	Describe("Load", func() {
+		It("loads shares including state and mountpoint information", func() {
+
+			sharesChan := make(chan *collaboration.Share)
+			receivedChan := make(chan sharespkg.ReceivedShareWithUser)
+
+			share := &collaboration.Share{
+				Id: &collaboration.ShareId{OpaqueId: "1iaeiae$vlcvlcvlc!pzbpzbpzb"},
+				// FIXME we may have to deal with importing existing share ids ... without a storage or provider prefix
```

Comment:
If this `FIXME` is important I would suggest moving it to the actual code, instead of a test file noone will be looking at until it breaks.

Reply:
can be dropped, updateShareID() implements this